common/examples.js

```diff
@@ -19,30 +19,39 @@ $(document).ready(function() {
 		return examples.findIndex(example => example.name == targetName)
 	}
 
-	function applyByFunctionFilter(functionName) {
-		var filterText = functionName.toLowerCase()
+	function applyByFunctionFilter(searchString) {
+		var filterText = searchString.toLowerCase()
 		if (filterText === "") {
 			$('#container').mixItUp('filter', exampleDivs);
 			$('#matches_counter').hide();
 			return
 		}
 		$('#matches_counter').show();
 
-		const filteredExamples = []
+		// using a set to avoid duplicates
+		const filteredExampleIndexes = new Set();
+
 		for (var functionName in functionUsages) {
 			if (!functionName.toLowerCase().includes(filterText)) continue;
 
 			for (var usage of functionUsages[functionName]) {
-				const exampleIndex = findExampleIndexByName(exampleData, usage)
-				if (!filteredExamples.includes(exampleIndex)) {
-					filteredExamples.push(exampleIndex)
-				}
+				const exampleIndex = findExampleIndexByName(exampleData, usage);
+				filteredExampleIndexes.add(exampleIndex);
+			}
+		}
+
+		// check all functions and all tags for any match
+		for (var functionName in examplesTags) {
+			const matchingTags = examplesTags[functionName].filter(tagItem => tagItem.includes(filterText));
+			if (matchingTags.length > 0) {
+				const exampleIndex = findExampleIndexByName(exampleData, functionName);
+				filteredExampleIndexes.add(exampleIndex);
```

Comment:
this does similar logic as the `functionName` searching but will add examples that have matching tags to the users input